transportable-udfs-documentation/transport-udfs-presto.patch

```diff
@@ -0,0 +1,139 @@
+diff --git a/presto-main/pom.xml b/presto-main/pom.xml
```

Comment:
Was this file included by accident?

Reply:
No. This is a patch that needs to be applied to the Presto engine to enable it to accept SqlScalarFunction classes as plugins. We create SqlScalarFunction presto UDFs.